pybossa/cache/projects.py

```diff
@@ -182,8 +188,11 @@ def search_lock_status_sorting_result():
 
         params["assign_user"] = args["sql_params"]["assign_user"]
 
-        all_available_tasks_sql = sql + filters +\
-                " ORDER BY %s" % (args.get('order_by') or 'priority_0 desc')
+        order_by =  args.get('order_by') or "priority_0 DESC"
```

Comment:
The code changes here will need to be careful with backwards compatibility in the SQL query. Feel free to add necessary testing (manual and unit tests) to ensure the other column sorting functions correctly.

Another option would be to create a separate method from `format_current_page()` that is specific to assigned_users.